docs/docs-content/clusters/edge/trusted-boot/deployment-day2/install.md

```diff
@@ -0,0 +1,179 @@
+---
+sidebar_label: "Installation"
+title: "Installation with Trusted Boot"
+description: "Learn about how to install Palette Edge with Trusted Boot on your Edge device."
+hide_table_of_contents: false
+sidebar_position: 50
+tags: ["edge"]
+---
+
+This page guides you through installing Palette Edge with Trusted Boot enabled on an Edge device. Some of the steps
+included in this guide involve the Basic Input/Output System (BIOS) interface of the Edge device which varies from
+machine to machine. This guide uses the Intel NUC 13 Pro as an example.
+
+## Prerequisites
+
+- Edge Installer ISO file. Check out the [Build Edge Artifacts with Trusted Boot](../edgeforge/build-trusted-iso.md) to
+  learn how to create an Edge Installer ISO with Trusted Boot.
+- The ability to modify the boot order settings to boot from a USB drive and to enter the BIOS interface. Typically this
+  requires you to have a wired keyboard and a monitor to display the interface connected to the Edge device.
+- A USB disk containing the installer ISO.
+
+:::warning
+
+Before proceeding with installation, check the maximum EFI size that your Edge host is able to boot and compare it
+against the installer ISO and provider image. With secure boot enabled and keys enrolled, you won't be able to boot from
+the EFI size checking ISO unless you disable secure boot or clear the keys. For more information, refer to
+[Check EFI Size and Edge Host Boot Limit](../edgeforge/check-efi-limit.md).
+
+:::
+
+## Instructions
+
+### Enroll Secure Boot Keys into Edge Device
```

Comment:
Back to back headings feels odd. Can we add some filler text or potentially remove one subheading?